src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -787,6 +787,13 @@ public SyntaxNode NamespaceImportDeclaration(string name)
             return NamespaceImportDeclaration(DottedName(name));
         }
 
+        /// <summary>
+        /// Creates a alias import declaration.
+        /// </summary>
+        /// <param name="aliasIdentifierName">The name of the alias.</param>
+        /// <param name="symbol">The namespace or type to be aliased.</param>
+        public abstract SyntaxNode AliasImportDeclaration(string aliasIdentifierName, INamespaceOrTypeSymbol symbol);
```

Comment:
❗️ This should have the following signature:

```csharp
public abstract SyntaxNode AliasImportDeclaration(string alias, SyntaxNode name);
```

If you want to keep a signature that operates on `INamespaceOrTypeSymbol`, you can use a second overload:

```csharp
public SyntaxNode AliasImportDeclaration(string alias, INamespaceOrTypeSymbol symbol) =>
    AliasImportDeclaration(alias, { see below });
```

The missing section of the helper method should call a method similar to `TypeExpression(ITypeSyntax)`, except it should forward the call to `GenerateNameSyntax` instead of `GenerateTypeSyntax`.